CONTRIBUTING.md

```diff
@@ -190,7 +190,7 @@ Run the tests by issuing the following command
 % bundle exec rspec spec/acceptance
 ```
 
-This will now download a pre-fabricated image configured in the [default node-set](./spec/acceptance/nodesets/default.yml),
```

Comment:
This file is managed by modsync.   These changes may be revertted.

Reply:
@ThoughtCrhyme, if this has been resolved, I think we're a go for merge. If not, how do we plan to manage possible overwrite by modsync?

Reply:
Yep, its going to get overwritten. Make the change here and created a PR for modsync